### PR TITLE
Add try-except if an empty list occurs and removed the exit function

### LINKED ISSUE #18: Error with last build probably some issue with last commit.
image

### PR DESCRIPTION
**Pull request by athul**
Closes #18

cc: @gautamkrishnar

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -45,10 +44,14 @@ def get_stats() -> str:
...
         print("Please Add your WakaTime API Key to the Repository Secrets")
-        sys.exit(1)
+        
 
     data_list = []
-    pad = len(max([l['name'] for l in lang_data[:5]], key=len))
+    try:
+        pad = len(max([l['name'] for l in lang_data[:5]], key=len))
+    except ValueError:
+        print("The Data seems to be empty. Please wait for a day for the data to be filled in.")
```
Comment (gautamkrishnar) [inline main.py:53]: https://github.com/athul/waka-readme/issues/18#issuecomment-662504219 Please check this comment, i think it is better, Just add a message to the readme. What do you think?

File: main.py [hunk 2]
```diff
@@ -45,10 +44,14 @@ def get_stats() -> str:
         lang_data = data['data']['languages']
     except KeyError:
         print("Please Add your WakaTime API Key to the Repository Secrets")
```
Comment (gautamkrishnar) [inline main.py:46]: Its okay to throw error here

File: main.py [hunk 3]
```diff
@@ -84,7 +87,6 @@ def generate_new_readme(stats: str, readme: str) -> str:
         repo = g.get_repo(f"{user}/{user}")
     except GithubException:
         print("Authentication Error. Try saving a GitHub Token in your Repo Secrets or Use the GitHub Actions Token, which is automatically used by the action.")
-        sys.exit(1)
```
Comment (gautamkrishnar) [inline main.py:87]: Here too

File: main.py (hunk 1 above)
Comment (athul) [reply main.py:53]: Message in the Readme is a bit too much I'd say. We don't want people visiting the profile get a bad impression that the data is missing.

Comment (athul) [reply main.py:53]: Or on 2nd thoughts, that will be better than to silently failing

Comment (gautamkrishnar) [reply main.py:53]: Then it's okay. Show message on the console and silent exit.

**Pull request merged**